bigtable/client/client_options.h

```diff
@@ -53,6 +53,36 @@ class ClientOptions {
     return *this;
   }
 
+  void SetCompressionAlgorithm(grpc::ChannelArguments *channel_arguments,
+                               const grpc_compression_algorithm algorithm);
+
+  void SetGrpclbFallbackTimeout(grpc::ChannelArguments *channel_arguments,
+                                const int fallback_timeout);
```

Comment:
Function arguments can't be mutable; does it make sense to mark integers as `const`? They're always by-value anyway.

I understand this means that you can no longer mutate this value even in the body of the function; I'm curious whether there's value to it, whether this is standard or not. Just asking.

Reply:
It is certainly allowed, but it has no effect and I dislike it.  Thanks for catching it.  Same thing for returning `const int` (or similar).

Reply:
Done.